tests/pages/components/course-page/repository-setup-help.js

```diff
@@ -0,0 +1,14 @@
+import { attribute, text } from 'ember-cli-page-object';
+
+export default {
+  linkGitCloneIssues: {
```

Comment:
```suggestion
  gitCloneIssuesLink: {
```

Most page object properties are named as <adjective><noun>, not the opposite. Examples: `continueButton`, `footerText` etc.